src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -1210,9 +1215,23 @@ private void EnsureFeatureManagementVersionInspected()
 
                 if (_requestTracingEnabled && _requestTracingOptions != null)
                 {
-                    _requestTracingOptions.FeatureManagementVersion = TracingUtils.GetAssemblyVersion(RequestTracingConstants.FeatureManagementAssemblyName);
+                    Version featureManagementVersion = TracingUtils.GetAssemblyVersion(RequestTracingConstants.FeatureManagementAssemblyName);
+
+                    Version featureManagementAspNetCoreVersion = TracingUtils.GetAssemblyVersion(RequestTracingConstants.FeatureManagementAspNetCoreAssemblyName);
+
+                    if (featureManagementVersion != null)
+                    {
+                        // If the version is less than 3.2.0, log the schema version warning
+                        if (featureManagementVersion.Major < 3 || (featureManagementVersion.Major == 3 && featureManagementVersion.Minor <= 1))
```

Comment:
`Version` class is `IComparable` which supports `<` operator. So it is possible to do this: 
``` C#
if (featureManagementVersion < new Version(3,2,0))
```
It is fine to keep the current implementation since we have comments to explain the `if` condition. Just another option for us.

Reply:
Thanks! I'll change to that